src/sentiment.jl

```diff
@@ -74,10 +78,22 @@ function Base.show(io::IO, s::SentimentAnalyzer)
 end
 
 
-function(m::SentimentModel)(text::Array{T, 1}) where T <: AbstractString
-    return get_sentiment(text, m.weight, m.words)
+function(m::SentimentModel)(text::Array{T, 1}, handle_unknown) where T <: AbstractString
```

Comment:
```suggestion
function (m::SentimentModel)(text::AbstractVector{<: AbstractString}, handle_unknown)
```

Also again `handle_unknown` should be first argument.